resources/views/mship/feedback/form.blade.php

```diff
@@ -20,7 +20,7 @@
 
         clearTimeout(timeout);
         timeout = window.setTimeout(function(){
-          $.get('{{route('mship.feedback.usersearch', null)}}/' + $('#member-search-name').val(), function(response){
+          $.get('{{route('mship.feedback.usersearch', /* error */ null)}}/' + $('#member-search-name').val(), function(response){
```

Comment:
@atoff This is erroring on load because `{name}` is a required param in the route. I'm not happy making it optional given the controller relies on it. Thoughts?